### ISSUE TITLE
If debug level is Warning, UI keeps saying "Synergy is starting"

### ISSUE THREAD

**Issue by silvester747** [symless/synergy#5605]
### Operating Systems

Client: OSX 10.11.6

Server: Ubuntu 14.04.5 LTS

### Synergy Version

synergy-v1.8.3-rc2-89c9c58

### Steps to reproduce bug

1. Make sure the server is not running (click 'Stop')
2. In the menu open 'Edit' -> 'Settings' and set 'Logging level' to 'Warning'
3. Click 'Start'

In the bottom left of the UI it still states "Synergy is starting."
If you change to any other log level (eg 'Info') it will change to "Synergy is running."

**Comment by silvester747** [comment]
I was originally thinking Synergy was not working at all, but then I found out a misconfiguration on the client side. So this message confuses a lot if you have a problem.

**Comment by nlyan** [comment]
Good catch! This is because of a nasty hack we use to detect the status of the core process.

**Comment by erikliland** [comment]
May there be some connection between this issue and #5408 ?

**Comment by nlyan** [comment]
@erikliland that's right. we will be removing these sorts of hacks in near future.
